### PR TITLE
added note about PDC emulator

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8094
Source: https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-faq#why-is-the-key-admins-group-missing-i-have-windows-server-2016-domain-controllers

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-dir-sync.md [hunk 1]
```diff
@@ -65,6 +65,9 @@ Sign-in a domain controller or management workstation with _Domain Admin_ equiva
 > [!NOTE]
 > If your AD forest has multiple domains, make sure you add the ADConnect sync service account (ie. MSOL_12121212) into "Enterprise Key Admins" group to gain permission across the domains in the forest.
 
+> [!NOTE]
+> Transfer the PDC emulator FSMO role to a domain controller running Windows Server 2016 (or later) to be able to search Key Admins and Enterprise Key Admins groups (domain controllers running previous versions of Windows Server cannot translate the security identifier to a name for these groups).
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-dir-sync.md:69]:
```suggestion
> Transfer the PDC emulator FSMO role to a domain controller running Windows Server 2016 (or later) to be able to search the Key Admins and Enterprise Key Admins groups (domain controllers running previous versions of Windows Server cannot translate the security identifier to a name for these groups).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-dir-sync.md (hunk 1 above)
Comment (e0i) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-dir-sync.md:69]: @MaratMussabekov Can you follow up with the copy editing suggestions? Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by mapalko — APPROVED** [review]

**Pull request merged**